### ISSUE TITLE
Freezes on Vs. Excitebike

### ISSUE THREAD

**Issue by litaniast** [SourMesen/Mesen#831]
Running on the libretro version of Mesen

When I select the difficulty and course, hangs on that screen and doesn't start.

![Vs  Excitebike (1988)(Nintendo)-200513-011741](https://user-images.githubusercontent.com/4857928/81719324-e4656e00-94b7-11ea-9348-1c51bc820daa.png)

**Comment by SourMesen** [comment]
There are multiple versions of this FDS dump, as far as I can tell. One of them freezes (CRC32: 0x69050650), and all the others seem to work properly. I'm not sure if this is a bad dump, or if this is actually an emulation problem, though.

**Comment by litaniast** [comment]
Sorry. I got another FDS file of Vs. Excitebike and it works well on Mesen.

**Comment by heckez-sys** [comment]
it could be a fan edit for the game or something? like some early romhacks for the nes dont work on newer emulators

**Comment by litaniast** [comment]
> it could be a fan edit for the game or something? like some early romhacks for the nes dont work on newer emulators

No. I tried the not hacked fds image from TOSEC. After that, I tried 'alternate' image, it works. strange.
